### ISSUE TITLE
Test for multi pod share pvc

### ISSUE THREAD

**Issue by woohhan** [woohhan/dropbox-csi#21]

**Comment by anencore94** [comment]
Is it possible to create dropbox-pv,pvc with 'RWM accessmode' and use such pvc in multi pods with current version?

**Comment by woohhan** [comment]
@anencore94 the node publish/node stage exsists for that purpose in the csi driver. I think it is possible if we apply a little modification.

**Comment by anencore94** [comment]
@woohhan Thanks for your reply. Is there any plan to provide such feature? It would be nice to have such feature for general users who are not familiar with remote file/obj-storage such as nfs, ceph-fs, etc..

**Comment by woohhan** [comment]
> @woohhan Thanks for your reply. Is there any plan to provide such feature? It would be nice to have such feature for general users who are not familiar with remote file/obj-storage such as nfs, ceph-fs, etc..

@anencore94 Of course, This issue is about to fix it. However, it will take some time due to my schedule...

**Comment by anencore94** [comment]
Thanks a lot !!! @woohhan
